src/sctools/test/test_fastq.py

```diff
@@ -188,3 +188,57 @@ def test_fields_populate_properly(reader_all_compressions):
         assert all(i in alphabet for i in record.sequence.strip())
         assert record.name2 == name2_string
         assert all(i in ascii_chars for i in record.quality.strip())
+
+
+# TEST BarcodeGeneratorWithCorrectedCellbarcodes
```

Comment:
Can you group these tests into a test class `TestBarcodeGeneratorWithCorrectedCellbarcodes`?

Reply:
Same answer as above.